examples/cookbook/regression/test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can we name this file regression_test.py (or regressions_test.py) instead? Our testing script looks for files matching *_test.py.

Reply:
Done